lib/rouge/lexers/shell.rb

```diff
@@ -9,10 +9,12 @@ class Shell < RegexLexer
 
       tag 'shell'
       aliases 'bash', 'zsh', 'ksh', 'sh'
-      filenames '*.sh', '*.bash', '*.zsh', '*.ksh',
-                '.bashrc', '.zshrc', '.kshrc', '.profile', 'APKBUILD', 'PKGBUILD'
+      filenames '*.sh', '*.bash', '*.zsh', '*.ksh', '.bashrc', '.zshrc',
+                '.kshrc', '.profile', 'APKBUILD', 'PKGBUILD', '*.ebuild',
+                '*.eclass', '*.exheres-0', '*.exlib'
 
-      mimetypes 'application/x-sh', 'application/x-shellscript'
+      mimetypes 'application/x-sh', 'application/x-shellscript', 'text/x-sh',
```

Comment:
When I went looking for the correct MIME type for shell scripts, I didn't find much but it looked like `text` was correct. Why did you prefer doing it this way?

Reply:
as the [commit message](https://github.com/rouge-ruby/rouge/pull/716/commits/761606f1c41f1752cedc4b06094252ce4020b649) says:

```
`file` utility uses `text/*` mime-types, while Mozilla uses `application/*`
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types:

![Screenshot_20190910_014620](https://user-images.githubusercontent.com/5360877/64573754-06708480-d35c-11e9-9794-fc3fd473140a.png)

Reply:
Ah, sweet. Sorry, missed that it was in there. 

One last thing: can you add tests for each of these MIME types to `spec/lexers/shell_spec.rb`?